plugins/Admin/admin.js

```diff
@@ -48,4 +49,41 @@ exports.uptime = {
 		}
 		msg.channel.send("**Uptime**: " + timestr);
 	}
-}
\ No newline at end of file
+}
+
+exports.userrole = {
+	usage: "<@user> <@role>",
+    description: "Used to toggle @role of specified @user",
+    process: function(bot,msg,arg){
+        var args = arg.split(" ");
+		var user, role;
+		if(msg.member.hasPermission("MANAGE_ROLES")){
```

Comment:
It would be nice to check if the bot has `MANAGE_ROLES` so that it can fail gracefully if it doesn't.

Reply:
could also do another try catch for it as well, or a quick change to add & to the if statement